bids-validator/bids_validator/rules/file_level_rules.json

```diff
@@ -526,7 +532,7 @@
   },
 
   "microscopy": {
-    "regexp": "^[\\/\\\\](sub-[a-zA-Z0-9]+)[\\/\\\\](?:(ses-[a-zA-Z0-9]+)[\\/\\\\])?microscopy[\\/\\\\](sub-[a-zA-Z0-9]+)(?:(_ses-[a-zA-Z0-9]+))?(?:_sample-[a-zA-Z0-9]+)(?:_acq-[a-zA-Z0-9]+)?(?:_stain-[a-zA-Z0-9]+)?(?:_run-[0-9]+)?(?:_chunk-[a-zA-Z0-9]+)?(((@@@_microscopy_type_@@@)(@@@_microscopy_ext_@@@))|(_photo\\.jpg))$",
+    "regexp": "^[\\/\\\\](sub-[a-zA-Z0-9]+)[\\/\\\\](?:(ses-[a-zA-Z0-9]+)[\\/\\\\])?microscopy[\\/\\\\](sub-[a-zA-Z0-9]+)(?:(_ses-[a-zA-Z0-9]+))?(?:_sample-[a-zA-Z0-9]+)(?:_acq-[a-zA-Z0-9]+)?(?:_stain-[a-zA-Z0-9]+)?(?:_run-[0-9]+)?(?:_chunk-[a-zA-Z0-9]+)?(((@@@_microscopy_type_@@@)(@@@_microscopy_ext_@@@))(@@@_photo_ext_@@@))$",
```

Comment:
Here we are going to have to split into 2 possible regex expressions, because the filename template for `_photo` is not the same as for the other suffix, from comment here: https://github.com/bids-standard/bids-validator/pull/1359#discussion_r759659733

1. For all the "microscopy_type" (TEM, SEM, uCT, etc):
```
 "regexp": "^[\\/\\\\](sub-[a-zA-Z0-9]+)[\\/\\\\](?:(ses-[a-zA-Z0-9]+)[\\/\\\\])?microscopy[\\/\\\\](sub-[a-zA-Z0-9]+)(?:(_ses-[a-zA-Z0-9]+))?(?:_sample-[a-zA-Z0-9]+)(?:_acq-[a-zA-Z0-9]+)?(?:_stain-[a-zA-Z0-9]+)?(?:_run-[0-9]+)?(?:_chunk-[a-zA-Z0-9]+)?(((@@@_microscopy_type_@@@)(@@@_microscopy_ext_@@@))$"
```
See reference here: https://bids-specification--881.org.readthedocs.build/en/881/04-modality-specific-files/10-microscopy.html#microscopy-imaging-data

2. For the `_photo` suffix:
```
"regexp": "^[\\/\\\\](sub-[a-zA-Z0-9]+)[\\/\\\\](?:(ses-[a-zA-Z0-9]+)[\\/\\\\])?microscopy[\\/\\\\](sub-[a-zA-Z0-9]+)(?:(_ses-[a-zA-Z0-9]+))?(?:_sample-[a-zA-Z0-9]+)(?:_acq-[a-zA-Z0-9]+)?(_photo)(@@@_photo_ext_@@@))$"
```

See reference here: https://bids-specification--881.org.readthedocs.build/en/881/04-modality-specific-files/10-microscopy.html#photos-of-the-samples-_photoextension

My syntax is probably not correct, let me know if you have questions.